### ISSUE TITLE
File size and last modified time?

### ISSUE THREAD

**Issue by justapps4u** [spacecowboy/NoNonsense-FilePicker#132]
Any way we can add the file size and last modified time below the file name?

**Comment by spacecowboy** [comment]
of course. just modify the layout suitably and populate it when you load the files.

See this example: 

* http://spacecowboy.github.io/NoNonsense-FilePicker/example/custom_item_layout/ 

as well as the image sample:
* https://github.com/spacecowboy/NoNonsense-FilePicker/tree/master/sample/src/main/java/com/nononsenseapps/filepicker/sample/multimedia
* https://github.com/spacecowboy/NoNonsense-FilePicker/blob/master/sample/src/main/res/layout/listitem_image_checkable.xml
